server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
+    },
+    answer: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Answer'
+    }
+})
+
+const TopicSchema = new Schema({
+    topic: {
+        type: String
+    }
+})
+
+const GameSchema = new Schema({
+    currentTopic: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Topic' },
+    round: {
+        type: Number
+    },
+    users: [UserSchema],
```

Comment:
so this would mean that we don't save users on their own and then connect them to a game, but that we save them only on the game. is that what we want? or do we want to have them as an independent collection and then have this be an array of references to the users collection?